container/sagemaker/tensorflow-serving.js

```diff
@@ -168,38 +168,29 @@ function json_lines_request(r, data) {
 function csv_request(r) {
     var data = r.requestBody
 
-    // look for initial quote or numeric-only data in 1st field
-    var needs_quotes = data.search(/^\s*("|[\d.Ee+\-]+\s*,)/) != 0
-
+    var needs_quotes = data.search(/^\s*(\[+?"|[\d.Ee+\-]+\s*,)/) != 0
```

Comment:
Input isn’t json why add the leading \[ ?

Reply:
whoops, that wasn't supposed to be there